hw05_parallel_execution/go.mod

```
@@ -1,4 +1,4 @@
-module github.com/fixme_my_friend/hw05_parallel_execution
+module github.com/kokdot/hw05_parallel_execution
```

module github.com/kokdot/hw05_parallel_execution
```suggestion
module github.com/kokdot/otus/hw05_parallel_execution
```